projects/xUnit/scripts/BasicHandlesTestSuite/BasicHandlesTestSuite.gml

```diff
@@ -0,0 +1,27 @@
+function BasicHandlesTestSuite() : TestSuite() constructor {
+    addFact("asset_reference_handles_test", function() {
```

Comment:
This could be a addTheory instead of a addFact (theories are exactly this... data driven tests)